api/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/containerd/containerd/api
+
+go 1.22.0
+
+require (
+	github.com/containerd/ttrpc v1.2.3
+	github.com/containerd/typeurl/v2 v2.1.1
+	github.com/opencontainers/image-spec v1.1.0
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20240401170217-c3f982113cda
+	google.golang.org/grpc v1.63.2
+	google.golang.org/protobuf v1.33.0
```

Comment:
Only thing worth considering is to see if those versions are required to be "current" version; some release branches may be on older image-spec and/or grpc; the API module should define minimum required version so that those branches can use the version they need (usually higher than minimum required version)